features/support/env.rb

```diff
@@ -51,6 +51,7 @@
 # of your scenarios, as this makes it hard to discover errors in your application.
 ActionController::Base.allow_rescue = false
 
+Cucumber::Rails::Database.javascript_strategy = :truncation, {except: %w[ar_internal_metadata]}
```

Comment:
Could you add a comment above for this temporary fix with a link to the issue / pull request in the database cleaner repo?

Reply:
Sure, done :)